internal/sinkcores/outputcore/override.go

```diff
@@ -88,10 +83,23 @@ func (c *overrideCore) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapco
 			continue
 		}
 
+		// If we found a matching override, use its level check
 		if o.Level.Enabled(ent.Level) {
 			return c.Core.Check(ent, ce)
 		}
+		return ce
```

Comment:
just so its more obvious, we return ce here since that means we won't log it? Not clear to me how this code works, I don't remember how this wrapping works :) But I assume that is the intention based on the o.Level.Enabled line.

Reply:
Yes I'm also not a 100% on how the wrapping context uses this, but experimentally, when `c.Core.Check(ent, ce)` is called, the entry will be logged (or at least it has passed this check), and when the original entry is returned it gets ignored.